src/state-summary/state-card-climate.html

```diff
@@ -52,14 +56,23 @@
       type: Boolean,
       value: false,
     },
-
     stateObj: {
       type: Object,
     },
   },
 
   computeTargetTemperature: function (stateObj) {
-    return stateObj.attributes.temperature + ' ' + stateObj.attributes.unit_of_measurement;
+    if (stateObj.attributes.operation_mode.includes('Auto')) {
```

Comment:
Please don't use `includes(…)`, it is part of ES2015 and will break all versions of Internet Explorer [docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes).

Use `.indexOf('Auto') != -1` instead.